test/minimal_init.lua

```diff
@@ -0,0 +1,71 @@
+local execute = vim.api.nvim_command
+local fn = vim.fn
+
+local install_path = "/tmp/site/pack/packer/opt/packer.nvim"
+
+if fn.empty(fn.glob(install_path)) > 0 then
+  execute("!git clone https://github.com/wbthomason/packer.nvim " .. install_path)
+end
+
+execute "packadd packer.nvim"
+
+vim.o.termguicolors = true
```

Comment:
Is this relevant? I know you're looking for fairly batteries included. But I don't see how this is related to LSP.

Reply:
What does the diagnostic highlighting look like without termcolors?

Reply:
highligthing would show neovim errors more than lspconfig errors. I would vote for removing this.

Reply:
Isn't the `default` theme just using 8 colors anyway?

Reply:
done